src/stylesheets/global.scss

```diff
@@ -87,7 +87,7 @@ button.btn.inverted, a.btn.inverted {
   }
 }
 
-// Overwrite the rule from opds-web-client that sets all top-level divs to 100% height. This works
+// Overwrite the rule from web-opds-client that sets all top-level divs to 100% height. This works
```

Comment:
I'm curious about why the name has changed around the words.

Reply:
@RishiDiwanTT This is because we renamed the project when we forked it. You can see it in the change on l. 62 of the README.

Reply:
Oh yes I was quite sure the rename was intentional I was just curious as to the motivation. This is a purely educational question (for myself) and has no bearing on the already approved PR.